index.js

```diff
@@ -15,7 +15,7 @@ const getWorker = (file, content, options) => {
     const fallbackWorkerPath = options.fallback === false ? 'null' : workerPublicPath;
     return `require(${createInlineWorkerPath})(${JSON.stringify(content)}, ${fallbackWorkerPath})`;
   }
-  return `new Worker(${workerPublicPath})`;
+  return `new Worker(${workerPublicPath} + window.location.search)`;

```

Comment:
`query | params | queryParams` but depending upon, if this could be used for other webworker path related stuff aswell, so it could be made more generic 😛. Needs triage :D